crates/network-scanner/src/mdns.rs

```diff
@@ -57,21 +57,21 @@ pub fn mdns_query_scan(
         task_manager
             .with_timeout(single_query_duration)
             .when_finish(move || {
-                tracing::debug!("Stopping browse for service: {}", service_name_clone);
+                tracing::debug!("stopping browse for service: {}", service_name_clone);
```

Comment:
style: For consistency, log messages should start with an uppercase, you don’t have to revert this. [See this](https://github.com/Devolutions/IronRDP/blob/39ca17b9c3b1cdd73ef533634f2e0950e3bf3bc1/STYLE.md#logging).

Reply:
updated